app/components/PrintedText.js

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+
+var PrintedText = function(props) {
+	return (
+		<div dangerouslySetInnerHTML={props.getMarkedText()}>
```

Comment:
This took me a long time to figure out because I didn't know about the name lol.